plugins/main/public/components/common/welcome/welcome.scss

```diff
@@ -45,7 +45,7 @@ span.statWithLink:hover {
   text-decoration: underline;
 }
 
-[name="OverviewWelcome"] .flex-redirect-app-links {
```

Comment:
**question**: there are some remaining styles with the selector `[name="OverviewWelcome"]` on
- plugins/main/public/styles/common.scss
- plugins/main/public/styles/theme/dark/index.dark.scss

Should they be adapted or can be removed?

Reply:
Solved in https://github.com/wazuh/wazuh-dashboard-plugins/pull/6594/commits/f22e354f610475555fb056b08ea3b5ceca715c91